client/src/components/BootstrapForms/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
I would call the folder as a Component. 
It is Input not form

Reply:
+ it would be great to add styles for error here and improve Register as there we have the same forms styles. But I believe it would be better if @khrystynapylypets do this.

Reply:
done for Login